Update instructions for using icon substitutions (#10489)
qgis/QGIS-Documentation: docs/documentation_guidelines/writing.rst

.. figure:: /static/common/logo.png
   :width: 10 em

Icon substitution
.................

You can add an image inside a text paragraph (e.g., as a tool icon).
To do so, you first need to create an alias (also named substitution),

> i.e., a reference name that will be used to invoke the icon.

Reviewer: ```suggestion
which is a reference name used to display the icon.
```
Perhaps a simpler wording…
